<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Luke Zappia" />


<title>Twitter coverage of the ABACBS 2018 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="ABACBS18.html">ABACBS18</a>
    </li>
    <li>
      <a href="COMBINE18.html">COMBINE18</a>
    </li>
    <li>
      <a href="ASHG18.html">ASHG18</a>
    </li>
    <li>
      <a href="gi2018.html">GI2018</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the ABACBS 2018 conference</h1>
<h4 class="author"><em>Luke Zappia</em></h4>
<h4 class="date"><em>2018-11-28 11:40:03</em></h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Hashtag</td>
<td align="left">#ABACBS2018</td>
</tr>
<tr class="even">
<td align="left">Conference</td>
<td align="left">ABACBS 2018 conference</td>
</tr>
<tr class="odd">
<td align="left">Time Zone</td>
<td align="left">Australia/Melbourne</td>
</tr>
<tr class="even">
<td align="left">Days</td>
<td align="left">c(“2018-11-26”, “2018-11-27”, “2018-11-28”)</td>
</tr>
<tr class="odd">
<td align="left">Accent colour</td>
<td align="left">#dd0c15</td>
</tr>
<tr class="even">
<td align="left">Pallete</td>
<td align="left">Set1</td>
</tr>
<tr class="odd">
<td align="left">k-core</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">Topics k</td>
<td align="left">6</td>
</tr>
<tr class="odd">
<td align="left">Bigram filter</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">Fixed</td>
<td align="left">FALSE</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the ABACBS 2018 conference conference. A total of 1133 tweets from 193 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/ABACBS18/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2018-11-26 - 2018-11-28 in the Australia/Melbourne timezone.</p>
<p><img src="figures/ABACBS18/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/ABACBS18/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/ABACBS18/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/ABACBS18/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/ABACBS18/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/ABACBS18/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/ABACBS18/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/ABACBS18/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/ABACBS18/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/ABACBS18/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>If you are interested in joining my lab for a post-doc in research bioinformatics have a chat to me or message me at #abacbs2018 #COMBINE18</p>
<a href="https://t.co/404MibRFnZ" class="uri">https://t.co/404MibRFnZ</a>
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#ABACBS2018 Award Winners - Early-career researcher <span class="citation">@BelindaPhipson</span> - Open science <span class="citation">@torstenseemann</span> - Professional bioinformatics Alex Garnham - Mid-career researcher David Lynn - Senior research fellow Mark Ragan
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Congratulations <span class="citation">@torstenseemann</span> for your open science award at #abacbs2018. Well deserved!
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Congratulations to <span class="citation">@BelindaPhipson</span> for winning the #abacbs2018 early career researcher award. There couldn’t be a more deserving winner #soproud
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Thanks <span class="citation">@BelindaPhipson</span> for mentioning <span class="citation">@scRNAtools</span>. Check it out here if you are interested <a href="https://t.co/EClP64fOU7" class="uri">https://t.co/EClP64fOU7</a>. Submissions and updates welcome! #abacbs2018
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 international keynote <span class="citation">@DunhamLab</span> “Drivers of aneuploidy and adaptation in yeast” <a href="https://t.co/46wfHZhqMC" class="uri">https://t.co/46wfHZhqMC</a>
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
rebecca_poulos
</td>
<td style="text-align:left;">
<p>Thanks #abacbs2018 for the chance to give a talk about our recent #mutational #signatures study. If you’d like to, you can read more here - just published last week in <span class="citation">@PLOSGenetics</span>:</p>
<a href="https://t.co/0OQZXam0h0" class="uri">https://t.co/0OQZXam0h0</a>
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
rebecca_poulos
</td>
<td style="text-align:left;">
<p>Learning about en masse competitions using barcoded yeast strains - gives 124,744 fitness measurements, from <span class="citation">@DunhamLab</span> at #abacbs2018</p>
<a href="https://t.co/MRDbRNpsUX" class="uri">https://t.co/MRDbRNpsUX</a>
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
<p><span class="citation">@theosysbio</span>: “Data resuscitation”</p>
That’s an apt description of what Bioinformaticians do when they they are not engaged in the experimental design, but rather are asked to save the experiment. #abacbs2018
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>BAZAM your Bams! Poster 22 #abacbs2018</p>
<a href="https://t.co/Yn7ufEky7O" class="uri">https://t.co/Yn7ufEky7O</a> <a href="https://t.co/3Cf7N0G1Vk" class="uri">https://t.co/3Cf7N0G1Vk</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/ABACBS18/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/ABACBS18/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Managed to get an Oshlack lab photo of the whole group at #abacbs2018 #loveit <a href="https://t.co/dmun9v1Ha7" class="uri">https://t.co/dmun9v1Ha7</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Congratulations to <span class="citation">@BelindaPhipson</span> for winning the #abacbs2018 early career researcher award. There couldn’t be a more deserving winner #soproud
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:left;">
BelindaPhipson
</td>
<td style="text-align:left;">
Feel honoured to receive the ECR award at #abacbs2018. Thank you to my amazing mentors <span class="citation">@AliciaOshlack</span> and Gordon Smyth for nominating me.
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:left;">
cabbagesofdoom
</td>
<td style="text-align:left;">
Much deserved #abacbs2018 open source award to <span class="citation">@torstenseemann</span> - I’ve annotated multiple organisms with Prokka, and I’m a fan!
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
<p><span class="citation">@theosysbio</span>: “Data resuscitation”</p>
That’s an apt description of what Bioinformaticians do when they they are not engaged in the experimental design, but rather are asked to save the experiment. #abacbs2018
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>If you are interested in joining my lab for a post-doc in research bioinformatics have a chat to me or message me at #abacbs2018 #COMBINE18</p>
<a href="https://t.co/404MibRFnZ" class="uri">https://t.co/404MibRFnZ</a>
</td>
<td style="text-align:right;">
33
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Congratulations <span class="citation">@torstenseemann</span> for your open science award at #abacbs2018. Well deserved!
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 international keynote <span class="citation">@DunhamLab</span> “Drivers of aneuploidy and adaptation in yeast” <a href="https://t.co/46wfHZhqMC" class="uri">https://t.co/46wfHZhqMC</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#ABACBS2018 Award Winners - Early-career researcher <span class="citation">@BelindaPhipson</span> - Open science <span class="citation">@torstenseemann</span> - Professional bioinformatics Alex Garnham - Mid-career researcher David Lynn - Senior research fellow Mark Ragan
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
JoeCursons
</td>
<td style="text-align:left;">
The #ABACBS2018 professional bioinformatics award given to Alex Garnham - well deserved as the powerhouse of the <span class="citation">@WEHI_research</span> bioinformatics service unit and the saviour of many <span class="citation">@WEHI_Postdocs</span> with data to analyse!
</td>
<td style="text-align:right;">
28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/ABACBS18/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/ABACBS18/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Apparently the conference program has the hashtag at #abacbs2018 … Best to go with that and not #ABACBS18 I think <a href="https://t.co/Wq3VdV2jSn" class="uri">https://t.co/Wq3VdV2jSn</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>my current TweetDeck tab is “#combine18 OR #combine2018 OR #abacbs18 OR #abacbs2018”</p>
maybe I should include the SNPs associated too 😂🤦‍♀️ <a href="https://t.co/iBtt6sLApf" class="uri">https://t.co/iBtt6sLApf</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/ABACBS18/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Managed to get an Oshlack lab photo of the whole group at #abacbs2018 #loveit <a href="https://t.co/dmun9v1Ha7" class="uri">https://t.co/dmun9v1Ha7</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 international keynote <span class="citation">@DunhamLab</span> “Drivers of aneuploidy and adaptation in yeast” <a href="https://t.co/46wfHZhqMC" class="uri">https://t.co/46wfHZhqMC</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
As the Professional Bioinformaticians Representative at <span class="citation">@abacbs</span> and a co-organiser of the <span class="citation">@RLadiesMelb</span>, I couldn’t be any prouder of our Prof Bioinfo Award winner 2018… Alex Garnham !! 🤗🎉💃🏼💪👏 Congrats lady! 😎👌 #abacbs2018 <a href="https://t.co/ABt02OY59r" class="uri">https://t.co/ABt02OY59r</a>
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
Congratulations <span class="citation">@_hollywhitfield</span> for winning the <span class="citation">@combine_au</span> Best Talk prize and for sharing it with us at #abacbs2018. A great talk indeed! 👌 And good luck with the upcoming PhD! <span class="citation">@WEHI_research</span> <a href="https://t.co/KeIkV4JTi4" class="uri">https://t.co/KeIkV4JTi4</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
missMethyl #woohoo #abacbs2018 <span class="citation">@JovMaksimovic</span> <span class="citation">@BelindaPhipson</span> <a href="https://t.co/I9g1pV6Gjg" class="uri">https://t.co/I9g1pV6Gjg</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Great to see the Illumina developing open source software #ABACBS2018 #byebyeblackbox <a href="https://t.co/GuD4iLys9i" class="uri">https://t.co/GuD4iLys9i</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 Day 2 international keynote from <span class="citation">@ctsa11</span> <a href="https://t.co/NGe43lPpKQ" class="uri">https://t.co/NGe43lPpKQ</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Charity Law is unwell, so won’t be presenting at #abacbs2018, but you can read about her work on exploring intron signal at <a href="https://t.co/ip7TMoN34f" class="uri">https://t.co/ip7TMoN34f</a>. On the upside, the morning session is now back on time :) Get well soon Charity! <a href="https://t.co/lBrXREjZ4e" class="uri">https://t.co/lBrXREjZ4e</a>
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Leming Shi tells us about MAQC/SEQC projects: huge collaboration that used RNA mixtures to compare gene expression technologies and analysis methods. A valuable resource for the research community. Extending to biomarker analysis at present - hoping to publish soon #abacbs2018 <a href="https://t.co/BN8LAV8I0C" class="uri">https://t.co/BN8LAV8I0C</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
Laura__Cook
</td>
<td style="text-align:left;">
Excited for my first #bioinformatics conference this week! #ABACBS2018 #phd #thylacine #genomics <span class="citation">@unimelb</span> <a href="https://t.co/4wuWT8zLQN" class="uri">https://t.co/4wuWT8zLQN</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<p><img src="http://pbs.twimg.com/media/Ds98dWvU0AA8xWT.jpg" /></p>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/ABACBS18/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/ABACBS18/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/ABACBS18/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Belinda Phipson discussing growing kidney organoids in a dish and assessing variability between these (kidney organoid maturity is a big contributor towards the variability). Single cell analysis shows highly variable genes, confirming maturity drives variability #ABACBS2018
</td>
<td style="text-align:right;">
0.9932673
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
CFlensburg
</td>
<td style="text-align:left;">
<p>#abacbs18 (aka #abacbs2018) going down this week, conference starting tomorrow, and I’ll try to live report a bit. Program here: <a href="https://t.co/CmbEoA7kJX" class="uri">https://t.co/CmbEoA7kJX</a></p>
<p>Feel free to follow/filter/unfollow/mute/block/report as you feel appropriate.</p>
For other live tweeps: <span class="citation">@TweetDeck</span> is awesome.
</td>
<td style="text-align:right;">
0.9932673
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@theosysbio</span>:⁩ mutual information, in particular partial information decomposition works better for network reconstruction. Nice results on neuronal differentiation published in Stumpf PS et al. (2017) <a href="https://t.co/0WxhlEoTsP" class="uri">https://t.co/0WxhlEoTsP</a> - first author is a long lost cousin :) #abacbs2018 <a href="https://t.co/M0qYvOvTr4" class="uri">https://t.co/M0qYvOvTr4</a>
</td>
<td style="text-align:right;">
0.9919338
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Improving mutual information (Mc Mahon et Al. J. Roy Soc. 2018), multivariate information measures, partial information decomposition. Switching from R to Julia gave a tremendous computational speed up… (not surprising) #abacbs2018
</td>
<td style="text-align:right;">
0.9919338
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
minouye271
</td>
<td style="text-align:left;">
<p>If anyone at #abacbs2018 is looking for a postdoc in multiomics or polygenic risk scores in Aus or UK, feel free to DM/email me.</p>
<p>The <span class="citation">@CamBakerSGI</span> is recruiting at both its <span class="citation">@BakerResearchAu</span> &amp;amp; <span class="citation">@Cambridge_Uni</span> nodes. Eg <a href="https://t.co/oJ3a6ytDBK" class="uri">https://t.co/oJ3a6ytDBK</a></p>
More info at <a href="https://t.co/R5AQWPf89N" class="uri">https://t.co/R5AQWPf89N</a>
</td>
<td style="text-align:right;">
0.9915136
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
BP: Batch to batch variability can be explained by relative maturity of organoids. This can be applied to a disease modelling experiment using a patient mutation and corrected line. #abacbs2018
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
LonsBio
</td>
<td style="text-align:left;">
Application of <span class="citation">@bphipson</span> variability results in patient corrected iPSC cells work from <span class="citation">@kidney_tom</span> et al, filtering out highly variable genes for DE analysis <a href="https://t.co/DjoBK5l47n" class="uri">https://t.co/DjoBK5l47n</a> #ABACBS2018
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Fantastic to hear from #combine18 oral presentation winner <span class="citation">@_hollywhitfield</span> today at #abacbs2018 i do want to note that all COMBINE speakers were at such a high calibre, really speaks to the talent and hard work of Australian bioinformatics students!
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
BP: iPS cells can be used to grow a mini kidney organoid. These can be used for disease modelling, drug screening etc. But how similar are they to a real kidney. #abacbs2018
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
As the Professional Bioinformaticians Representative at <span class="citation">@abacbs</span> and a co-organiser of the <span class="citation">@RLadiesMelb</span>, I couldn’t be any prouder of our Prof Bioinfo Award winner 2018… Alex Garnham !! 🤗🎉💃🏼💪👏 Congrats lady! 😎👌 #abacbs2018 <a href="https://t.co/ABt02OY59r" class="uri">https://t.co/ABt02OY59r</a>
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Taiyun Kim is speaking about choice of similarity metric (e.g. Pearson/Spearman correlation? Euclidean?) and how it affects single cell clustering. Found essentially that correlation based metrics perform better at recovering cell types #abacbs2018
</td>
<td style="text-align:right;">
0.9932673
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
Taiyun Kim explains how #kmeans clustering performed better when using correlation-based similarity measures (Pearson and Spearman) than when distance-based ones were used considering pre-defined labels. #confBingo #abacbs2018 <a href="https://t.co/lZLQXG0w31" class="uri">https://t.co/lZLQXG0w31</a>
</td>
<td style="text-align:right;">
0.9915136
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
KevinWang009
</td>
<td style="text-align:left;">
Taiyun Kim from <span class="citation">@sydneybioinfo</span> group on similarity metrics for single cell RNA-Seq data at #abacbs2018. A surprising result: Correlation based metrics tend to do better than distances metrics like Euclidean distance!
</td>
<td style="text-align:right;">
0.9915136
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>Conference Bingo for #abacbs2018 is here! Download your card and start tweeting using the #confBingo hashtag. Post you winning row and winning complete card for a bottle of wine on me!</p>
<p><a href="https://t.co/TsHcgYWIiU" class="uri">https://t.co/TsHcgYWIiU</a></p>
Thanks <span class="citation">@LonsBio</span> <span class="citation">@BreonSchmidt</span> #ABACBS18
</td>
<td style="text-align:right;">
0.9915136
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
sydneybioinfo
</td>
<td style="text-align:left;">
Several <span class="citation">@sydneybioinfo</span> members are in attendance at #abacbs2018 with talks + posters. Give these people a follow &amp;amp; come say Hi! <span class="citation">@KevinWang009</span> Taiyun Kim (talk today), Hani Kim, <span class="citation">@LinYingxin</span> <span class="citation">@rylmb1</span> <span class="citation">@shazanfar</span> <span class="citation">@TheEllisPatrick</span> Nick Canete, Heeva Baharlou, Kitty Lo, Dario Strbenac
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>Richard Edwards on promise of 10x Chromium’s Diploid Phasing to perform comparable to long-read technologies at the cost of short read ones by #demultiplexing using #SNPs.</p>
#abacbs2018 <a href="https://t.co/ZPinB1tqFB" class="uri">https://t.co/ZPinB1tqFB</a>
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
One more 5 min student talk before the break - Taiyun Kim, “Impact of similarity metrics on single-cell RNA-seq data clustering” #abacbs2018
</td>
<td style="text-align:right;">
0.9892797
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Charity Law is unwell, so won’t be presenting at #abacbs2018, but you can read about her work on exploring intron signal at <a href="https://t.co/ip7TMoN34f" class="uri">https://t.co/ip7TMoN34f</a>. On the upside, the morning session is now back on time :) Get well soon Charity! <a href="https://t.co/lBrXREjZ4e" class="uri">https://t.co/lBrXREjZ4e</a>
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
kizza_a
</td>
<td style="text-align:left;">
hey <span class="citation">@CFlensburg</span> didnt get to ask at qanda. do you have to be careful with cnv on RNAseq if the samples were pooled? got caught out by pooled mice calling variants on RNAseq #abacbs2018
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
TK: Correlation based distance metrics lead to scRNA-seq clustering that better matches predefined labels #abacbs2018
</td>
<td style="text-align:right;">
0.9866425
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Maitreya Dunham on rich genomic datasets and thinking about CNV (causative? drivers?) in human disease (aneuploidy, cancer, complex disease). Yeast as a relevant system owing to CNVs affecting many traits in yeast, can perform experimental evolution in a chemostat #abacbs2018
</td>
<td style="text-align:right;">
0.9940100
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
cabbagesofdoom
</td>
<td style="text-align:left;">
<span class="citation">@koadman</span> wrt your repeat % Q: v. low gaps levels but also smaller assemblies for the other snake genomes analysed. Will need to dig deeper to tell missing data from possible 10x repeat over-assembly artefact. Are we repeating repeats? #ABACBS2018 #QandA
</td>
<td style="text-align:right;">
0.9923144
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
_hollywhitfield
</td>
<td style="text-align:left;">
<p>To account for individual variability, and administer the correct interventions Kim-An Do is producing integrated networks:</p>
<p>Interaction databases + data -&amp;gt; patient-specific networks</p>
#abacbs2018 #ABACBS18
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
methylnick
</td>
<td style="text-align:left;">
Twas a brilliant talk and controls are fundamental to any interpretation of genomic data. It’s a slow adoption but my colleagues are starting to listen. #abacbs2018 #reproducibility #controls #referenceMaterial <a href="https://t.co/49SqJRvO0v" class="uri">https://t.co/49SqJRvO0v</a>
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Slides for my talk in this sessionhttps://speakerdeck.com/lazappi/visualising-trees-to-choose-clusters-for-scrna-seq-data. Software on CRAN <a href="https://t.co/EFeMPtEt3p" class="uri">https://t.co/EFeMPtEt3p</a> and GitHub <a href="https://t.co/sbmRKgygP8" class="uri">https://t.co/sbmRKgygP8</a> #abacbs2018
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 A big issue with comparing cell proportions with droplet scRNA-Seq data is the need to make correct selection of high quality cells. I demonstrate this in my poster, presenting tomorrow <a href="https://t.co/4bXD2Pejy3" class="uri">https://t.co/4bXD2Pejy3</a>
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
JoeCursons
</td>
<td style="text-align:left;">
The #ABACBS2018 professional bioinformatics award given to Alex Garnham - well deserved as the powerhouse of the <span class="citation">@WEHI_research</span> bioinformatics service unit and the saviour of many <span class="citation">@WEHI_Postdocs</span> with data to analyse!
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Twitter analysis from #COMBINE18 yesterday <a href="https://t.co/a41UHNAMYR" class="uri">https://t.co/a41UHNAMYR</a>. I will be doing the same for #ABACBS2018. Tweet a link to GitHub, CRAN or Bioconductor to add something to the software list.
</td>
<td style="text-align:right;">
0.9876548
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
Pumped to be chairing the Genomics session this afternoon at #ABACBS2018 Great start with an inspiring talk from Prof Leming Shi from Shanghai 😎👌 <a href="https://t.co/hu4EALcL9Q" class="uri">https://t.co/hu4EALcL9Q</a>
</td>
<td style="text-align:right;">
0.9876548
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
frostickle
</td>
<td style="text-align:left;">
Afternoon keynote by Leming Shi at #ABACBS2018 - Quality Control and Standardization of Omics and Bioinformatics for Precision Medicine. <a href="https://t.co/ldOdCbPqMJ" class="uri">https://t.co/ldOdCbPqMJ</a>
</td>
<td style="text-align:right;">
0.9866425
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Leming Shi tells us about MAQC/SEQC projects: huge collaboration that used RNA mixtures to compare gene expression technologies and analysis methods. A valuable resource for the research community. Extending to biomarker analysis at present - hoping to publish soon #abacbs2018 <a href="https://t.co/BN8LAV8I0C" class="uri">https://t.co/BN8LAV8I0C</a>
</td>
<td style="text-align:right;">
0.9937813
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Stephen J. Bent has a story of four moths and working on ancient DNA samples (characteristic damage would be lots of cytosine deamination, base oxidation of purines, shorter fragments). One of these moth samples is almost 100 years old! C&amp;gt;T &amp;amp; G&amp;gt;A at ends is the hint! #abacbs2018
</td>
<td style="text-align:right;">
0.9926606
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 hearing from <span class="citation">@S_Foroutan</span> on singscore, method to estimate <em>truly</em> single sample gene set/pathway scores. These are useful for building interpretable and stable features for cohort-independent and prospective learning.
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
DunhamLab
</td>
<td style="text-align:left;">
I’m up first this morning at #abacbs2018! I’ll talk about “Drivers of aneuploidy and adaptation in yeast” focusing on a few datasets we’ve generated that I’d love to set this crowd loose on for more analysis.
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@DunhamLab</span>⁩ uses a pooled approach to gather fitness measures in yeast and associate with changes in copy number using regression. Fun fact: barcoding pioneered by yeast geneticists in the ‘90s - now used by everyone #abacbs2018 <a href="https://t.co/idw6ib6mqc" class="uri">https://t.co/idw6ib6mqc</a>
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
paulfharrison
</td>
<td style="text-align:left;">
.@CFlensburg describing SuperFreq. Parallel plots of copy number and variant allele frequency. Both can be segmented, but it works better to do both simultaneously. A nice simple example of multi-task learning, maybe. #abacbs2018
</td>
<td style="text-align:right;">
0.9892797
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 Ralph Patrick describing Differential Proportion Testing for cell types across sample conditions using scRNA-seq, during and after myocardial infarction
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
carolyndg
</td>
<td style="text-align:left;">
Presenting a paper at #ABACBS2018 on <a href="https://t.co/yIWi4gzkMV" class="uri">https://t.co/yIWi4gzkMV</a> with new Haemopedia RNA-seq 🐭 &amp;amp;👩 haematopoeitic gene expression datasets. Come and chat about lineage signatures and data portals.
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Take aways from <span class="citation">@cabbagesofdoom</span> #abacbs2018 talk: Linked reads look useful for assembly, it’s possible to assemble a species in a few months if you have a course to run, be careful about interpreting BUSCO results 🐍
</td>
<td style="text-align:right;">
0.9876548
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@rebecca_poulos</span>⁩ gives us a fantastic talk on linking mutational signatures and driver mutations in cancer - this work was recently published! <a href="https://t.co/cS8CiiRKtj" class="uri">https://t.co/cS8CiiRKtj</a> #abacbs2018 <a href="https://t.co/yS1AIPHTBm" class="uri">https://t.co/yS1AIPHTBm</a>
</td>
<td style="text-align:right;">
0.9876548
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@DunhamLab</span>⁩ improved upon results from DNAcopy to segment data along the genome in collaboration with statisticians ⁦<span class="citation">@daniela_witten</span>⁩ and Lucy Gao. Applied 1D Fused Lasso to help distinguish between ‘steppy’ or linear models and map driver genes #abacbs2018 <a href="https://t.co/UdxjsF4j23" class="uri">https://t.co/UdxjsF4j23</a>
</td>
<td style="text-align:right;">
0.9932673
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 Kim-Anh Do now speaking about PRECISE, Personalised cancer-specific integrated estimation. Two main aims in personalised medicine: build more accurate understanding of individual patients, and introduce adaptive treatments to individuals
</td>
<td style="text-align:right;">
0.9926606
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
paulfharrison
</td>
<td style="text-align:left;">
<p>.@dunhamlab looking at driver genes vs larger regions of fitness of genes with 1D fused LASSO at #abacbs2018 (if I’m understanding correctly).</p>
Hearing LASSO these days I wonder if Elastic-Net can be used, here would give intermediates between stepwise and smooth curve fit.
</td>
<td style="text-align:right;">
0.9923144
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
** Shaming Tweet Warning ** With over 50 people identifying as Professional Bioinformaticians in Australia, and most of them being from Melbourne… How come it was only 5 of us at the Prof Bioinfo get together tonight?? 🤔 Never mind, more food &amp;amp; drinks for us! 😜 #abacbs2018 <a href="https://t.co/2W6euJ4JNz" class="uri">https://t.co/2W6euJ4JNz</a>
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
jakebradfordqut
</td>
<td style="text-align:left;">
I heard from <span class="citation">@LGFearnley</span> in today’s poster session at #ABACBS2018. It was interesting to hear the computational challenges he is facing around designing pgRNA for #CRISPR screening experiments. Thanks for your time, <span class="citation">@LGFearnley</span>.
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
YiwenWang_Eva
</td>
<td style="text-align:left;">
Glad that people are interested in how to select methods to adjust for batch effect, even though not for microbiome data. I got many good questions and suggestions today! #abacbs2018 <span class="citation">@tpq__</span> <span class="citation">@annaquagli</span> <span class="citation">@mritchieau</span> Thanks for the picture from my friends! <span class="citation">@youyupei</span> <span class="citation">@jiaan_yu</span> <a href="https://t.co/WlsNTT6gCx" class="uri">https://t.co/WlsNTT6gCx</a>
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
lnly0311
</td>
<td style="text-align:left;">
Chris Saunders giving a talk about Improving sequence analysis to increase the clinical value of whole genome sequencing. #abacbs2018 - “Not using JAVA” - people clapping
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
Professional Bioinformaticians attending #abacbs2018 (#abacbs18) Join us for the social event at the Shaw Davey Slum after the <span class="citation">@abacbs</span> opening ceremony tonight! <a href="https://t.co/CmWQ4RMjAO" class="uri">https://t.co/CmWQ4RMjAO</a>
</td>
<td style="text-align:right;">
0.9876548
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>The “inevitable tSNE plot” possibly distinguishing the tumour cells from others - from Dr Ann-Marrie Patch.</p>
#abacbs2018 <a href="https://t.co/oQGZcemJnU" class="uri">https://t.co/oQGZcemJnU</a>
</td>
<td style="text-align:right;">
0.9866425
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
<span class="citation">@ctsa11</span>: There is a lot of room for false positives with structural variant calling because with two breakpoints the search space is the size of the genome squared #abacbs2018
</td>
<td style="text-align:right;">
0.9866425
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Yi Jin Liew with epigenetic adaptation of corals to ocean acidification. Corals are very important to oceanic ecosystems, so relevant to know effects of rising temperatures etc. on coral. Chances are in humans, a random CpG site will be methylated. Coral, not so! #abacbs2018
</td>
<td style="text-align:right;">
0.9929771
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#ABACBS2018 Award Winners - Early-career researcher <span class="citation">@BelindaPhipson</span> - Open science <span class="citation">@torstenseemann</span> - Professional bioinformatics Alex Garnham - Mid-career researcher David Lynn - Senior research fellow Mark Ragan
</td>
<td style="text-align:right;">
0.9915136
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
frostickle
</td>
<td style="text-align:left;">
<p>Bioinformatics career panel time at #COMBINE18 with: <span class="citation">@IBMResearch</span> <span class="citation">@bwgoudey</span> <span class="citation">@PeterMacCC</span> <span class="citation">@RoxaneLegaie</span> <span class="citation">@illumina</span> <span class="citation">@ctsa11</span> <span class="citation">@QIMRBerghofer</span> <span class="citation">@IamAMP</span></p>
<p>Good to see a balance in industry vs public hospital vs academic &amp;amp; also gender!</p>
#ABACBS2018 <a href="https://t.co/pzB2UfR6zw" class="uri">https://t.co/pzB2UfR6zw</a>
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>Leming Shi explains how lucky we are working in life sciences with a lot of things to discover compared to drug developers who have a rough path ahead until it a real difference is made. #PrecisionMedicine</p>
#abacbs2018 <a href="https://t.co/9JAH8eFHXE" class="uri">https://t.co/9JAH8eFHXE</a>
</td>
<td style="text-align:right;">
0.9910472
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Yi Jin Liew: coral CpG methylation is very different to patterns seen in mammals. Mostly occurs in gene bodies and much rarer (~ 7%) at CpG islands compared to humans (~ 60-80%) #abacbs2018 <a href="https://t.co/PjWS0noQZa" class="uri">https://t.co/PjWS0noQZa</a>
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
There are several reasons why rare disease cases remain undiagnosed: 1) variant of unknown significance (lots of work happening here) 2) variants we still don’t have tools (methods, technology)to detect 3) not genetic My lab is interested in 2) #abacbs2018
</td>
<td style="text-align:right;">
0.9905265
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 <span class="citation">@steman_research</span> presenting work on ARMET (algorithm for resolving microenvironment transcriptomes). Passes on hierarchical structure of cell types rather than treating independently to deconvolute bulk transcriptomes
</td>
<td style="text-align:right;">
0.9899416
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Ann-Marie plots walking us through the challenges of cancer genomics, different types of variants, tumour heterogeneity, challenges related to therapy. #abacbs2018
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@theosysbio</span> discusses the results of Marbach et al. (2012) from the DREAM 5 challenge. We must not use correlation to reconstruct gene regulatory networks is one of their take home messages. Who knew! <a href="https://t.co/XVNasvRh9S" class="uri">https://t.co/XVNasvRh9S</a> #abacbs2018
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
S_Foroutan
</td>
<td style="text-align:left;">
<span class="citation">@JoeCursons</span> is talking about silencing of tumour suppressor genes through #methylation in liver cancer. Really loved that he mentioned all the tools he used to get the results 👍 #abacbs18 #abacbs2018 <a href="https://t.co/LUIHhMWnCT" class="uri">https://t.co/LUIHhMWnCT</a>
</td>
<td style="text-align:right;">
0.9885245
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">10</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, Bioconductor or CRAN.</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">abseqPy</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/malhamdoosh/abseqpy" class="uri">https://github.com/malhamdoosh/abseqpy</a></td>
</tr>
<tr class="even">
<td align="left">abseqR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/abseqR" class="uri">https://bioconductor.org/packages/abseqR</a></td>
</tr>
<tr class="odd">
<td align="left">badvisualisations</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/biovisml/badvisualisations" class="uri">https://bitbucket.org/biovisml/badvisualisations</a></td>
</tr>
<tr class="even">
<td align="left">bazam</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ssadedin/bazam" class="uri">https://github.com/ssadedin/bazam</a></td>
</tr>
<tr class="odd">
<td align="left">bionix</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/papenfusslab/bionix" class="uri">https://github.com/papenfusslab/bionix</a></td>
</tr>
<tr class="even">
<td align="left">canvas</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/canvas" class="uri">https://github.com/illumina/canvas</a></td>
</tr>
<tr class="odd">
<td align="left">celaref</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/monashbioinformaticsplatform/celaref" class="uri">https://github.com/monashbioinformaticsplatform/celaref</a></td>
</tr>
<tr class="even">
<td align="left">celaref</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/celaref" class="uri">https://bioconductor.org/packages/celaref</a></td>
</tr>
<tr class="odd">
<td align="left">clustree</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lazappi/clustree" class="uri">https://github.com/lazappi/clustree</a></td>
</tr>
<tr class="even">
<td align="left">clustree</td>
<td align="left">CRAN</td>
<td align="left"><a href="https://CRAN.R-project.org/package=clustree" class="uri">https://CRAN.R-project.org/package=clustree</a></td>
</tr>
<tr class="odd">
<td align="left">crisscrosslinker</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/egmg726/crisscrosslinker" class="uri">https://github.com/egmg726/crisscrosslinker</a></td>
</tr>
<tr class="even">
<td align="left">dee2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/markziemann/dee2" class="uri">https://github.com/markziemann/dee2</a></td>
</tr>
<tr class="odd">
<td align="left">EndoVir</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ncbi-hackathons/endovir" class="uri">https://github.com/ncbi-hackathons/endovir</a></td>
</tr>
<tr class="even">
<td align="left">ExpansionHunter</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/expansionhunter" class="uri">https://github.com/illumina/expansionhunter</a></td>
</tr>
<tr class="odd">
<td align="left">GeneMates</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/wanyuac/genemates" class="uri">https://github.com/wanyuac/genemates</a></td>
</tr>
<tr class="even">
<td align="left">havic</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/schultzm/havic" class="uri">https://github.com/schultzm/havic</a></td>
</tr>
<tr class="odd">
<td align="left">manta</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/manta" class="uri">https://github.com/illumina/manta</a></td>
</tr>
<tr class="even">
<td align="left">Mixture_model_feature_selection</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/marcelacespedes/mixture_model_feature_selection" class="uri">https://github.com/marcelacespedes/mixture_model_feature_selection</a></td>
</tr>
<tr class="odd">
<td align="left">paragraph</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/paragraph" class="uri">https://github.com/illumina/paragraph</a></td>
</tr>
<tr class="even">
<td align="left">parktree</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tchappell/parktree" class="uri">https://github.com/tchappell/parktree</a></td>
</tr>
<tr class="odd">
<td align="left">Pathway-Visualiser</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/algarnham/pathway-visualiser" class="uri">https://github.com/algarnham/pathway-visualiser</a></td>
</tr>
<tr class="even">
<td align="left">plyranges</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sa-lee/plyranges" class="uri">https://github.com/sa-lee/plyranges</a></td>
</tr>
<tr class="odd">
<td align="left">Polaris</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/polaris" class="uri">https://github.com/illumina/polaris</a></td>
</tr>
<tr class="even">
<td align="left">Scavenger</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vccri/scavenger" class="uri">https://github.com/vccri/scavenger</a></td>
</tr>
<tr class="odd">
<td align="left">scClustBench</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/taiyunkim/scclustbench" class="uri">https://github.com/taiyunkim/scclustbench</a></td>
</tr>
<tr class="even">
<td align="left">SigClust</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tchappell/sigclust" class="uri">https://github.com/tchappell/sigclust</a></td>
</tr>
<tr class="odd">
<td align="left">singscore</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/singscore" class="uri">https://bioconductor.org/packages/singscore</a></td>
</tr>
<tr class="even">
<td align="left">strelka</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/strelka" class="uri">https://github.com/illumina/strelka</a></td>
</tr>
<tr class="odd">
<td align="left">STRetch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/oshlack/stretch#boomboomtsch" class="uri">https://github.com/oshlack/stretch#boomboomtsch</a></td>
</tr>
<tr class="even">
<td align="left">superFreq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/christofferflensburg/superfreq" class="uri">https://github.com/christofferflensburg/superfreq</a></td>
</tr>
<tr class="odd">
<td align="left">superFreq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/christofferflensburg/superfreq/blob/master/readme.md" class="uri">https://github.com/christofferflensburg/superfreq/blob/master/readme.md</a></td>
</tr>
<tr class="even">
<td align="left">trndiff</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/biovisml/trndiff.2015" class="uri">https://bitbucket.org/biovisml/trndiff.2015</a></td>
</tr>
<tr class="odd">
<td align="left">vsearch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/torognes/vsearch" class="uri">https://github.com/torognes/vsearch</a></td>
</tr>
<tr class="even">
<td align="left">working_with_dna_meth</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lyijin/working_with_dna_meth" class="uri">https://github.com/lyijin/working_with_dna_meth</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>##  setting  value                       
##  version  R version 3.5.1 (2018-07-02)
##  system   x86_64, darwin15.6.0        
##  ui       X11                         
##  language (EN)                        
##  collate  en_AU.UTF-8                 
##  tz       Australia/Melbourne         
##  date     2018-11-28                  
## 
##  package      * version date       source        
##  assertthat     0.2.0   2017-04-11 CRAN (R 3.5.0)
##  backports      1.1.2   2017-12-13 CRAN (R 3.5.0)
##  base         * 3.5.1   2018-07-05 local         
##  bindr          0.1.1   2018-03-13 CRAN (R 3.5.0)
##  bindrcpp     * 0.2.2   2018-03-29 CRAN (R 3.5.0)
##  broom          0.5.0   2018-07-17 CRAN (R 3.5.0)
##  cli            1.0.0   2017-11-05 CRAN (R 3.5.0)
##  colorspace     1.3-2   2016-12-14 CRAN (R 3.5.0)
##  compiler       3.5.1   2018-07-05 local         
##  crayon         1.3.4   2017-09-16 CRAN (R 3.5.0)
##  curl           3.2     2018-03-28 CRAN (R 3.5.0)
##  datasets     * 3.5.1   2018-07-05 local         
##  devtools       1.13.6  2018-06-27 cran (@1.13.6)
##  digest         0.6.16  2018-08-22 CRAN (R 3.5.0)
##  dplyr        * 0.7.6   2018-06-29 cran (@0.7.6) 
##  evaluate       0.11    2018-07-17 CRAN (R 3.5.0)
##  fansi          0.3.0   2018-08-13 CRAN (R 3.5.0)
##  forcats      * 0.3.0   2018-02-19 CRAN (R 3.5.0)
##  ggforce        0.1.3   2018-07-07 CRAN (R 3.5.0)
##  ggplot2      * 3.0.0   2018-07-03 CRAN (R 3.5.0)
##  ggraph       * 1.0.2   2018-07-07 CRAN (R 3.5.0)
##  ggrepel      * 0.8.0   2018-05-09 cran (@0.8.0) 
##  glue           1.3.0   2018-07-17 CRAN (R 3.5.0)
##  graphics     * 3.5.1   2018-07-05 local         
##  grDevices    * 3.5.1   2018-07-05 local         
##  grid           3.5.1   2018-07-05 local         
##  gridExtra      2.3     2017-09-09 CRAN (R 3.5.0)
##  gtable         0.2.0   2016-02-26 CRAN (R 3.5.0)
##  here         * 0.1     2017-05-28 CRAN (R 3.5.0)
##  highr          0.7     2018-06-09 CRAN (R 3.5.0)
##  htmltools      0.3.6   2017-04-28 CRAN (R 3.5.0)
##  httr           1.3.1   2017-08-20 CRAN (R 3.5.0)
##  igraph       * 1.2.2   2018-07-27 CRAN (R 3.5.0)
##  janeaustenr    0.1.5   2017-06-10 CRAN (R 3.5.0)
##  jsonlite       1.5     2017-06-01 CRAN (R 3.5.0)
##  knitr        * 1.20    2018-02-20 CRAN (R 3.5.0)
##  labeling       0.3     2014-08-23 CRAN (R 3.5.0)
##  lattice        0.20-35 2017-03-25 CRAN (R 3.5.1)
##  lazyeval       0.2.1   2017-10-29 CRAN (R 3.5.0)
##  lubridate    * 1.7.4   2018-04-11 CRAN (R 3.5.0)
##  magrittr       1.5     2014-11-22 CRAN (R 3.5.0)
##  MASS           7.3-51  2018-10-16 CRAN (R 3.5.0)
##  Matrix         1.2-14  2018-04-13 CRAN (R 3.5.1)
##  memoise        1.1.0   2017-04-21 CRAN (R 3.5.0)
##  methods      * 3.5.1   2018-07-05 local         
##  modeltools     0.2-22  2018-07-16 CRAN (R 3.5.0)
##  munsell        0.5.0   2018-06-12 cran (@0.5.0) 
##  nlme           3.1-137 2018-04-07 CRAN (R 3.5.1)
##  NLP            0.1-11  2017-08-15 CRAN (R 3.5.0)
##  openssl        1.0.2   2018-07-30 CRAN (R 3.5.0)
##  parallel       3.5.1   2018-07-05 local         
##  pillar         1.3.0   2018-07-14 CRAN (R 3.5.0)
##  pkgconfig      2.0.2   2018-08-16 CRAN (R 3.5.0)
##  plyr           1.8.4   2016-06-08 CRAN (R 3.5.0)
##  purrr        * 0.2.5   2018-05-29 CRAN (R 3.5.0)
##  R6             2.2.2   2017-06-17 CRAN (R 3.5.0)
##  RColorBrewer * 1.1-2   2014-12-07 CRAN (R 3.5.0)
##  Rcpp           0.12.18 2018-07-23 CRAN (R 3.5.0)
##  reshape2       1.4.3   2017-12-11 CRAN (R 3.5.0)
##  rlang          0.2.2   2018-08-16 CRAN (R 3.5.0)
##  rmarkdown      1.10    2018-06-11 CRAN (R 3.5.0)
##  rprojroot      1.3-2   2018-01-03 CRAN (R 3.5.0)
##  rtweet       * 0.6.7   2018-06-20 CRAN (R 3.5.0)
##  scales         1.0.0   2018-08-09 CRAN (R 3.5.0)
##  slam           0.1-43  2018-04-23 CRAN (R 3.5.0)
##  SnowballC      0.5.1   2014-08-09 CRAN (R 3.5.0)
##  stats        * 3.5.1   2018-07-05 local         
##  stats4         3.5.1   2018-07-05 local         
##  stringi        1.2.4   2018-07-20 CRAN (R 3.5.0)
##  stringr      * 1.3.1   2018-05-10 CRAN (R 3.5.0)
##  tibble         1.4.2   2018-01-22 CRAN (R 3.5.0)
##  tidyr        * 0.8.1   2018-05-18 CRAN (R 3.5.0)
##  tidyselect     0.2.4   2018-02-26 CRAN (R 3.5.0)
##  tidytext     * 0.1.9   2018-05-29 CRAN (R 3.5.0)
##  tm             0.7-5   2018-07-29 CRAN (R 3.5.0)
##  tokenizers     0.2.1   2018-03-29 CRAN (R 3.5.0)
##  tools          3.5.1   2018-07-05 local         
##  topicmodels  * 0.2-7   2017-11-03 CRAN (R 3.5.0)
##  tweenr         0.1.5   2016-10-10 CRAN (R 3.5.0)
##  units          0.6-0   2018-06-09 CRAN (R 3.5.0)
##  utf8           1.1.4   2018-05-24 CRAN (R 3.5.0)
##  utils        * 3.5.1   2018-07-05 local         
##  viridis      * 0.5.1   2018-03-29 CRAN (R 3.5.0)
##  viridisLite  * 0.3.0   2018-02-01 cran (@0.3.0) 
##  withr          2.1.2   2018-03-15 CRAN (R 3.5.0)
##  wordcloud    * 2.6     2018-08-24 CRAN (R 3.5.0)
##  xml2           1.2.0   2018-01-24 CRAN (R 3.5.0)
##  yaml           2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
