<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Luke Zappia" />


<title>Twitter coverage of the 2019 Keystone Symposia Single Cell Biology conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="KSSingleCell18.html">KSSingleCell19</a>
    </li>
    <li>
      <a href="ABACBS18.html">ABACBS18</a>
    </li>
    <li>
      <a href="COMBINE18.html">COMBINE18</a>
    </li>
    <li>
      <a href="ASHG18.html">ASHG18</a>
    </li>
    <li>
      <a href="gi2018.html">GI2018</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the 2019 Keystone Symposia Single Cell Biology conference</h1>
<h4 class="author"><em>Luke Zappia</em></h4>
<h4 class="date"><em>2019-01-21 09:25:25</em></h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Hashtag</td>
<td align="left">#KSSingleCell</td>
</tr>
<tr class="even">
<td align="left">Conference</td>
<td align="left">Keystone Single Cell</td>
</tr>
<tr class="odd">
<td align="left">Time Zone</td>
<td align="left">America/Denver</td>
</tr>
<tr class="even">
<td align="left">Days</td>
<td align="left">c(“2019-01-13”, “2019-01-14”, “2019-01-15”, “2019-01-16”, “2019-01-17”)</td>
</tr>
<tr class="odd">
<td align="left">Accent colour</td>
<td align="left">#0d204a</td>
</tr>
<tr class="even">
<td align="left">Pallete</td>
<td align="left">Set1</td>
</tr>
<tr class="odd">
<td align="left">k-core</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">Topics k</td>
<td align="left">6</td>
</tr>
<tr class="odd">
<td align="left">Bigram filter</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">Fixed</td>
<td align="left">TRUE</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Keystone Single Cell conference. A total of 647 tweets from 300 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/KSSingleCell18/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-01-13 - 2019-01-17 in the America/Denver timezone.</p>
<p><img src="figures/KSSingleCell18/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/KSSingleCell18/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/KSSingleCell18/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/KSSingleCell18/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/KSSingleCell18/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/KSSingleCell18/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/KSSingleCell18/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/KSSingleCell18/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/KSSingleCell18/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/KSSingleCell18/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
satijalab
</td>
<td style="text-align:left;">
Takeaways from amazing #KSsinglecell: 1. Spatial technologies are mature, and are tremendously exciting 2. Integrating scRNA-seq data with cell morphology, location, and functional readouts is key for interpretation 3. Single cell genomics is transforming developmental biology
</td>
<td style="text-align:right;">
126
</td>
</tr>
<tr>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Starting off the Single Cell Computational Biology session of the #KSSingleCell conference with a talk by <span class="citation">@fabian_theis</span>, “Modeling Differentiation and Stimulation Response in Single-Cell Genomics.” The five methods below were presented.
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
<span class="citation">@BartDeplancke</span> stuns the audience by introducing Live-Seq: Profiling single cell RNA WITHOUT(!!!) killing the cell. #KSsinglecell
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
How do we know if difference between samples are batch effects of biological differences? Have to agree with <span class="citation">@satijalab</span>: biological replicates are more important than numbers of cells. #KSsinglecell
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
Lots of great talks first day of #KSsinglecell (massive data sets, exciting new analyses), but Allon Klein’s BioRxiv paper (<a href="https://t.co/t6xkBEJ5XH" class="uri">https://t.co/t6xkBEJ5XH</a>) on fate prediction seems like a must read and very much liked the elegant lineage tracing work by <span class="citation">@morris_lab</span> using CellTag (1/2)
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
AlejoFraticelli
</td>
<td style="text-align:left;">
#KSsinglecell Keystone Single Cell Biology 2019 was a blast! My recap: 1) scRNAseq increasingly combined with other single cell technologies 2) scProteomics is almost here 3) scOmics data integration 4) We have a data storage problem 5) Visualization tools need much improvement
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
franschrandez
</td>
<td style="text-align:left;">
<p>Excited to be going to <span class="citation">@KeystoneSymp</span> and presenting Clustergrammer2 Jupyter widget at poster #1058 <span class="citation">@SinaiImmunol</span> <span class="citation">@10xgenomics</span> <span class="citation">@ProjectJupyter</span> <span class="citation">@mybinderteam</span> #KSsinglecell #scRNAseq #regl #webgl #python</p>
view notebook: <a href="https://t.co/X4XvJdFMRv" class="uri">https://t.co/X4XvJdFMRv</a> run notebook: <a href="https://t.co/oB2Kfynwqe" class="uri">https://t.co/oB2Kfynwqe</a> <a href="https://t.co/NQ7hiCvlUp" class="uri">https://t.co/NQ7hiCvlUp</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
rcannood
</td>
<td style="text-align:left;">
The presentation by <span class="citation">@Zouters</span> and me is available at <a href="https://t.co/YHT4lFf4Nn" class="uri">https://t.co/YHT4lFf4Nn</a> #KSsinglecell An older version of the benchmark is available at <a href="https://t.co/T13GUw3yhm" class="uri">https://t.co/T13GUw3yhm</a>, but an updated version will be available very soon!
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
inespatop
</td>
<td style="text-align:left;">
Great talk by <span class="citation">@fabian_theis</span>. They are creating amazing tools for #singlecell. Some of them already used by some friends in this paper: <a href="https://t.co/8ITRWp8IPD" class="uri">https://t.co/8ITRWp8IPD</a>. Go check them <a href="https://t.co/gYz4ogHxAL" class="uri">https://t.co/gYz4ogHxAL</a>. #KSsinglecell
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
I’ll be presenting our study of molecular pathology in ALS tonight (poster 3004) at #KSsinglecell. In it we use Spatial Transcriptomics (now part of ⁦<span class="citation">@10xgenomics</span>⁩) to profile expression in ~1300 mouse tissue sections and 80 PM sections. Stop by! <a href="https://t.co/DI7mEtLqjh" class="uri">https://t.co/DI7mEtLqjh</a>
</td>
<td style="text-align:right;">
7
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/KSSingleCell18/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/KSSingleCell18/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
satijalab
</td>
<td style="text-align:left;">
Takeaways from amazing #KSsinglecell: 1. Spatial technologies are mature, and are tremendously exciting 2. Integrating scRNA-seq data with cell morphology, location, and functional readouts is key for interpretation 3. Single cell genomics is transforming developmental biology
</td>
<td style="text-align:right;">
352
</td>
</tr>
<tr>
<td style="text-align:left;">
fabian_theis
</td>
<td style="text-align:left;">
Excited to talk at the Keystone meeting on Single Cell Biology (hosted by Bertie Göttgens and Timm Schroeder) starting now <span class="citation">@Breckenridge</span>. 400 people, fully booked, wow. #KSsinglecell
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Starting off the Single Cell Computational Biology session of the #KSSingleCell conference with a talk by <span class="citation">@fabian_theis</span>, “Modeling Differentiation and Stimulation Response in Single-Cell Genomics.” The five methods below were presented.
</td>
<td style="text-align:right;">
49
</td>
</tr>
<tr>
<td style="text-align:left;">
franschrandez
</td>
<td style="text-align:left;">
<p>Excited to be going to <span class="citation">@KeystoneSymp</span> and presenting Clustergrammer2 Jupyter widget at poster #1058 <span class="citation">@SinaiImmunol</span> <span class="citation">@10xgenomics</span> <span class="citation">@ProjectJupyter</span> <span class="citation">@mybinderteam</span> #KSsinglecell #scRNAseq #regl #webgl #python</p>
view notebook: <a href="https://t.co/X4XvJdFMRv" class="uri">https://t.co/X4XvJdFMRv</a> run notebook: <a href="https://t.co/oB2Kfynwqe" class="uri">https://t.co/oB2Kfynwqe</a> <a href="https://t.co/NQ7hiCvlUp" class="uri">https://t.co/NQ7hiCvlUp</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
How do we know if difference between samples are batch effects of biological differences? Have to agree with <span class="citation">@satijalab</span>: biological replicates are more important than numbers of cells. #KSsinglecell
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
<span class="citation">@BartDeplancke</span> stuns the audience by introducing Live-Seq: Profiling single cell RNA WITHOUT(!!!) killing the cell. #KSsinglecell
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
Lots of great talks first day of #KSsinglecell (massive data sets, exciting new analyses), but Allon Klein’s BioRxiv paper (<a href="https://t.co/t6xkBEJ5XH" class="uri">https://t.co/t6xkBEJ5XH</a>) on fate prediction seems like a must read and very much liked the elegant lineage tracing work by <span class="citation">@morris_lab</span> using CellTag (1/2)
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:left;">
AlejoFraticelli
</td>
<td style="text-align:left;">
#KSsinglecell Keystone Single Cell Biology 2019 was a blast! My recap: 1) scRNAseq increasingly combined with other single cell technologies 2) scProteomics is almost here 3) scOmics data integration 4) We have a data storage problem 5) Visualization tools need much improvement
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
sunnyz_wu
</td>
<td style="text-align:left;">
RAGE-SEQ at #KSsinglecell ! <span class="citation">@ManuSinghNZ</span> from <span class="citation">@GarvanInstitute</span> describing a powerful scRNA-Seq + targeted long read seq approach with many applications in tumour immunology. <a href="https://t.co/yADHoTGeSP" class="uri">https://t.co/yADHoTGeSP</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
fabian_theis
</td>
<td style="text-align:left;">
Mind-blowing talk by <span class="citation">@priscaliberali</span> at #KSsinglecell measuring &amp;gt;500k intestinal organoids and quantifying heterogeneity during their development - amongst others using pseudotemporal ordering, originally developed for single-cell data.
</td>
<td style="text-align:right;">
34
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/KSSingleCell18/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/KSSingleCell18/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Great work on benchmarking #singlecell trajectory methods. Discussed by <span class="citation">@Zouters</span> and <span class="citation">@rcannood</span> at #KSsinglecell. Great to see a collaborative effort including New Zealand researchers <span class="citation">@ppgardne</span>. <a href="https://t.co/PWjgstru7n" class="uri">https://t.co/PWjgstru7n</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
sciencyjessie
</td>
<td style="text-align:left;">
Great and usefull tool to explore methods to infer trajecories from for example scRNA-seq data. Great presentation by <span class="citation">@Zouters</span> and <span class="citation">@rcannood</span> at #KSsinglecell ! <a href="https://t.co/0U83og208i" class="uri">https://t.co/0U83og208i</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
There are &amp;gt;75 (!!!) methods for trajectory inference in scRNA-seq data. Great talk by <span class="citation">@rcannood</span> at #KSsinglecell on their excellent work comparing methods. Benchmarking existing methods can be tedious, but it is utterly crucial for practitioners who have no idea what to use. <a href="https://t.co/p4BgDi34oH" class="uri">https://t.co/p4BgDi34oH</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Interesting and fast solution for batch effect correction: Harmony. <span class="citation">@ilyakorsunsky</span> and <span class="citation">@soumya_boston</span> at #KSsinglecell. Compatible with Seurat #singlecell analysis workflows. <a href="https://t.co/oqiz26pa6f" class="uri">https://t.co/oqiz26pa6f</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Highly scalable method for correction of batch effect, based on soft clustering. 500,000 cells in about an hour. Great talk at #KSSingleCell by <span class="citation">@ilyakorsunsky</span>! There’s been a great emphasis methods addressing the problem of dataset integration at this conference. <a href="https://t.co/AaOXcTfww9" class="uri">https://t.co/AaOXcTfww9</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Great to hear about latest developments from <span class="citation">@ReikLab</span> at #KSsinglecell. Using the #epigenetic landscape applied at the #singlecell level has great potential towards a deeper understanding of developmental biology. <a href="https://t.co/b4heZEWpnX" class="uri">https://t.co/b4heZEWpnX</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
VictorGenet
</td>
<td style="text-align:left;">
Amazing method and results from the <span class="citation">@ReikLab</span> presented today at the #KSsinglecell scNMTseq = RNAseq + DNAmethylome + Chromatin accessibility 😲 I’d love to try that with neuron differentiation !!! <a href="https://t.co/0ZGiOxC1LC" class="uri">https://t.co/0ZGiOxC1LC</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/KSSingleCell18/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
franschrandez
</td>
<td style="text-align:left;">
<p>Excited to be going to <span class="citation">@KeystoneSymp</span> and presenting Clustergrammer2 Jupyter widget at poster #1058 <span class="citation">@SinaiImmunol</span> <span class="citation">@10xgenomics</span> <span class="citation">@ProjectJupyter</span> <span class="citation">@mybinderteam</span> #KSsinglecell #scRNAseq #regl #webgl #python</p>
view notebook: <a href="https://t.co/X4XvJdFMRv" class="uri">https://t.co/X4XvJdFMRv</a> run notebook: <a href="https://t.co/oB2Kfynwqe" class="uri">https://t.co/oB2Kfynwqe</a> <a href="https://t.co/NQ7hiCvlUp" class="uri">https://t.co/NQ7hiCvlUp</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
sunnyz_wu
</td>
<td style="text-align:left;">
RAGE-SEQ at #KSsinglecell ! <span class="citation">@ManuSinghNZ</span> from <span class="citation">@GarvanInstitute</span> describing a powerful scRNA-Seq + targeted long read seq approach with many applications in tumour immunology. <a href="https://t.co/yADHoTGeSP" class="uri">https://t.co/yADHoTGeSP</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
RongxinFang
</td>
<td style="text-align:left;">
Finished my first keystone presentation #KSSingleCell present SnapATAC for the first time. Now it’s time for skiing! #singlecell #atacseq <a href="https://t.co/wXqsmYnz9d" class="uri">https://t.co/wXqsmYnz9d</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
VincentCroset
</td>
<td style="text-align:left;">
Warming up (or cooling down?) for #KSsinglecell with <span class="citation">@ScottishWaddell</span> #FFP and Charlie Treiber. Looking forward to a nice conference in sunny #Breckenridge #Colorado <a href="https://t.co/UfVXIifbGQ" class="uri">https://t.co/UfVXIifbGQ</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
10xgenomics
</td>
<td style="text-align:left;">
We’re enjoying the Keystone Single Cell conference this week. Stop by our table or visit our Poster #3011, High Throughput Immune Profiling Using Integrated Single Cell Multi-omics Analyses. #KSsinglecell <a href="https://t.co/VRNiCvznyb" class="uri">https://t.co/VRNiCvznyb</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
MaehrLab
</td>
<td style="text-align:left;">
Looking forward to a great conference on Single Cell Biology. Scientific talks will start tomorrow. For now: View from my room #KSsinglecell <span class="citation">@KeystoneSymp</span> 👍 <a href="https://t.co/LfbnBFsFIL" class="uri">https://t.co/LfbnBFsFIL</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
ebfabyanic
</td>
<td style="text-align:left;">
Really happy to be here at #KSSingleCell 2019 in Breckenridge! I’m going to be giving a poster tonight (poster number 1054), and on Thursday, I’ll be giving a short talk during the “Novel Single Cell Technologies” workshop. Please stop by! I appreciate any and all feedback! 🧬 <a href="https://t.co/QXh4Zd31gd" class="uri">https://t.co/QXh4Zd31gd</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
sunnyz_wu
</td>
<td style="text-align:left;">
Poppin my <span class="citation">@KeystoneSymp</span> cherry this week at #KSsinglecell ! Abstracts this year look amazing. Come check out poster 4047 on thurs to see our groups work on breast cancer heterogeneity ! (In between the slopes) <a href="https://t.co/8uEgIyjTVB" class="uri">https://t.co/8uEgIyjTVB</a>
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
money_gupta
</td>
<td style="text-align:left;">
Country count for single cell 2019, some interesting stats!! <span class="citation">@KeystoneSymp</span> #KSsinglecell <a href="https://t.co/7Lj8gaNgAb" class="uri">https://t.co/7Lj8gaNgAb</a>
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
AlejoFraticelli
</td>
<td style="text-align:left;">
Best breckenridge 2 am food ever. #KSsinglecell <a href="https://t.co/JZ3YMFU88T" class="uri">https://t.co/JZ3YMFU88T</a>
</td>
<td style="text-align:right;">
11
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<p><img src="http://pbs.twimg.com/ext_tw_video_thumb/1083924236325015552/pu/img/EICH6KV1c12pnPbC.jpg" /></p>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/KSSingleCell18/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/KSSingleCell18/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/KSSingleCell18/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
Third #KSsinglecell recap: <span class="citation">@seeliglab</span> democratizes Split-seq with kit production, <span class="citation">@LucaRappez</span> &amp;gt;100 metabolite detection across 4k single cells, astonishing imaging by <span class="citation">@AllenInstitute</span> with great GFP-tagged iPSC line resource, joint scRNAseq &amp;amp; TF binding assay (Rob Mitra lab)
</td>
<td style="text-align:right;">
0.9962652
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
<p>Tracking proteolysis in live single cells using a fluorescent timer from <span class="citation">@davidsuter_epfl</span> at #KSsinglecell. Degradation of proteins pauses in mitosis. Half-lives of proteins co-vary within cells. Protein degradation and synthesis rates correlate in cells.</p>
<a href="https://t.co/REPbd4zo5v" class="uri">https://t.co/REPbd4zo5v</a>
</td>
<td style="text-align:right;">
0.9953368
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
satijalab
</td>
<td style="text-align:left;">
Takeaways from amazing #KSsinglecell: 1. Spatial technologies are mature, and are tremendously exciting 2. Integrating scRNA-seq data with cell morphology, location, and functional readouts is key for interpretation 3. Single cell genomics is transforming developmental biology
</td>
<td style="text-align:right;">
0.9953368
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Whats the diff between cell type and state? Hongkui Zeng from <span class="citation">@AllenInstitute</span> studies (dis-)continuous variation in scRNAseq. Adding modalities (morphology) improves ability to make the distinction. Common theme at #KSsinglecell: Go beyond transcriptome to understand a cell!
</td>
<td style="text-align:right;">
0.9951352
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
fabian_theis
</td>
<td style="text-align:left;">
Great talk at #KSsinglecell by Thomas Höfer <span class="citation">@DKFZ</span>, showing how to integrate barcodes into models of population dynamics. Super-interesting to combine with molecular readouts from scRNAseq, since many effects are seen as shifts of cell numbers per cell type cluster.
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Highly scalable method for correction of batch effect, based on soft clustering. 500,000 cells in about an hour. Great talk at #KSSingleCell by <span class="citation">@ilyakorsunsky</span>! There’s been a great emphasis methods addressing the problem of dataset integration at this conference. <a href="https://t.co/AaOXcTfww9" class="uri">https://t.co/AaOXcTfww9</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
Lots of great talks first day of #KSsinglecell (massive data sets, exciting new analyses), but Allon Klein’s BioRxiv paper (<a href="https://t.co/t6xkBEJ5XH" class="uri">https://t.co/t6xkBEJ5XH</a>) on fate prediction seems like a must read and very much liked the elegant lineage tracing work by <span class="citation">@morris_lab</span> using CellTag (1/2)
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
<span class="citation">@satijalab</span>: Data integration across modalities (scRNAseq, scATACseq, imaging) identifies subtle cellular phenotypes not observed using a single modality. Potentially necessary to capture true full biology of single cells? #KSsinglecell
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Amazing talks, great scientific intx. Personal takeaway: Computation &amp;amp; tech, really drive the field. Go beyond single modality/omic &amp;amp; come up w creative perturbations. And the view wasnt so bad either. #KSsinglecell <a href="https://t.co/3lowmmWVYi" class="uri">https://t.co/3lowmmWVYi</a>
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Starting off the Single Cell Computational Biology session of the #KSSingleCell conference with a talk by <span class="citation">@fabian_theis</span>, “Modeling Differentiation and Stimulation Response in Single-Cell Genomics.” The five methods below were presented.
</td>
<td style="text-align:right;">
0.9930247
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
bibaswanghoshal
</td>
<td style="text-align:left;">
The concept of Cell fate: probabilities of reaching the different terminal states of cells is interesting from <span class="citation">@dana_peer</span>!! But even more intriguing is the concept of probabilities over pseudotime! Can both co-exist in determining cellular lineages? #KSsinglecell
</td>
<td style="text-align:right;">
0.9930247
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Great talk/poster at #KSsinglecell. They use “matrix-free spectral clustering” which avoids forming the Laplacian: the left singular vectors of a row-normalized matrix correspond to the eigenvectors of the Laplacian defined using cosine similarity between all points. <a href="https://t.co/hP5K7KMYQh" class="uri">https://t.co/hP5K7KMYQh</a>
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
Beautiful talk by <span class="citation">@ReikLab</span>: #singlecell triple-omics (chromatin accessibility, DNA methylation, RNA expression) &amp;amp;analyses with Multi­Omics Factor Analysis (<a href="https://t.co/3Qc7QCeqGF" class="uri">https://t.co/3Qc7QCeqGF</a>) show that during gastrulation lineages are defined by a hierarchical epigenetic model #KSSinglecell
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Great first day at #KSsinglecell. Proud to be here to represent Japan and <span class="citation">@riken_en</span>. Looking forward to great Science tomorrow. Enjoying my first <span class="citation">@KeystoneSymp</span> meeting so far. Stunning clear skies in Colorado. Hope to meet more talented scientists and learn new tricks this week.
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
There are &amp;gt;75 (!!!) methods for trajectory inference in scRNA-seq data. Great talk by <span class="citation">@rcannood</span> at #KSsinglecell on their excellent work comparing methods. Benchmarking existing methods can be tedious, but it is utterly crucial for practitioners who have no idea what to use. <a href="https://t.co/p4BgDi34oH" class="uri">https://t.co/p4BgDi34oH</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
Schwartastic
</td>
<td style="text-align:left;">
Amazing talks and posters at #KSsinglecell! Excited to present TooManyCells poster today, #3057. TooManyCells is a program with a suite of tools including a new kind of visualization for cell clades! Check out the website at <a href="https://t.co/d3Zw8yXFpX" class="uri">https://t.co/d3Zw8yXFpX</a> and our bioRxiv paper!
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
.@MaehrLab CRISPRi-perturbation screen with scRNA-seq readout for analysing loss-of-function phenotypes of candidate regulators. New insights into regulation of human endoderm development #KSSinglecell #singlecell #CRISPR
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
arnavm1
</td>
<td style="text-align:left;">
The poster is up! If you want to hear about how you–yes you!–can simultaneously do scRNA-seq and identify transcription factor binding sites, come check it out. I’ll be at poster 3018 from 7:30-10 PM MST tonight #KSsinglecell #callingcards #SRT
</td>
<td style="text-align:right;">
0.9934318
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
<p>Great points for Timm Schroeder (ETH) at #KSsinglecell.</p>
<ul>
<li><p>#singlecell analysis is great but we must be careful interpreting a snapshot</p></li>
<li>researchers must be computationally literate to tackle novel problems and ensure analysis is doing what it should be doing. Time well spent.
</td>
<td style="text-align:right;">
0.9930247
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Decontx: a Bayesian method for correcting for background noise (floating RNA) in #singlecell data. Available as an R package. Shiyi Yang (Boston University) at #KSsinglecell
</td>
<td style="text-align:right;">
0.9925638
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
A conservative approach to imputing drop outs in #singlecell data: Adaptively-thresholded Low-Rank Approximation (ALRA). #KSsinglecell <a href="https://t.co/vztobWqxKB" class="uri">https://t.co/vztobWqxKB</a>
</td>
<td style="text-align:right;">
0.9920377
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Lots of progress in #singlecell #epigenomics at #KSsinglecell. Bisulfate treatment and ACE-Seq enables accurate 5hmC sequencing. <span class="citation">@ebfabyanic</span> and colleagues at Penn State are optimising for single cell analysis. #epigenetics <a href="https://t.co/p99K96wdxY" class="uri">https://t.co/p99K96wdxY</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
10xgenomics
</td>
<td style="text-align:left;">
We’re enjoying the Keystone Single Cell conference this week. Stop by our table or visit our Poster #3011, High Throughput Immune Profiling Using Integrated Single Cell Multi-omics Analyses. #KSsinglecell <a href="https://t.co/VRNiCvznyb" class="uri">https://t.co/VRNiCvznyb</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
Very cool method (SpaceM) just presented by <span class="citation">@luca_rappez</span> at #KSsinglecell. By combining MALDI-IMS with imaging in a clever way, they are able to relate spatially resolves metabolite levels back to single cells and gain functional insight. Ideas percolating on how we could use it!
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
bibaswanghoshal
</td>
<td style="text-align:left;">
The view from the window! Excited for the Keystone Conference starting tomorrow here <span class="citation">@beaverrunresort</span>! #KSsinglecell #conference #snow #rockies #travelgram #traveldiaries2019 #breckenridge #mountains #snowymountains #viewfrommywindow <a href="https://t.co/HiMmhMEKWX" class="uri">https://t.co/HiMmhMEKWX</a> <a href="https://t.co/RyhQ0hN5L1" class="uri">https://t.co/RyhQ0hN5L1</a>
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
sciencyjessie
</td>
<td style="text-align:left;">
Excited to hear great science and talk science at #KSsinglecell meeting in Breckenridge Colorado. Want to know more about immuno-detection by sequencing for intracellular phosphoprotein measurements and how to couple Abs to DNA tags, come and see me at poster 4027
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
L Rappez from <span class="citation">@alexandrovteam</span> <span class="citation">@thalexandrov</span> <span class="citation">@embl</span> SpaceM: method for in situ spatial single-cell #metabolomics, linking #singlecell metabolomes (MALDI analysis) with spatial and morphological features (microscopy &amp;amp;histology) #KSSinglecell
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
FLOW-MAP: a graph-based algorithm for #singlecell time course experiments with CyTOF or #scRNASeq. A graphical interface supports interaction with the data. Corey Williams (University of Virginia) #KSsinglecell
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Had an amazing time at the #KSsinglecell meeting. Many great speakers and good overview of trends in the field but what I’m really inspired by is the abundance of talented and ambitious young researchers. #singlecell analysis is the future. The future is in good hands.
</td>
<td style="text-align:right;">
0.9937939
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
clec9Andrew
</td>
<td style="text-align:left;">
Excited to be at <span class="citation">@KeystoneSymp</span> #KSsinglecell! Stop by poster 2048 (session 2) for some data &amp;amp; discussion on lung cancer immune micro-environment, CITEseq, and all of the batch-robust analyses that we’re doing with <span class="citation">@effiken</span> in the <span class="citation">@MiriamMerad</span> lab <span class="citation">@SinaiImmunol</span>!
</td>
<td style="text-align:right;">
0.9937939
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
AlejoFraticelli
</td>
<td style="text-align:left;">
Heading to Breckenridge, CO, for a wonderful Keystone symposium on Single Cell Biology. Excited to show our work on hematopoiesis at the single cell level!!!!!! #KSSingleCell #singlecell #HSCI #stemcell #hematology <span class="citation">@KeystoneSymp</span>
</td>
<td style="text-align:right;">
0.9934318
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
fabian_theis
</td>
<td style="text-align:left;">
Excited to talk at the Keystone meeting on Single Cell Biology (hosted by Bertie Göttgens and Timm Schroeder) starting now <span class="citation">@Breckenridge</span>. 400 people, fully booked, wow. #KSsinglecell
</td>
<td style="text-align:right;">
0.9934318
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Caroline Uhler shows off nice trick to ‘translate’ scRNAseq methodology (optimal transport (OT)) into imaging: Apply OT algorithm in latent space derived from variational autoencoder to overcome coordinate system inconsistency in imaging data #KSsinglecell
</td>
<td style="text-align:right;">
0.9955223
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
2/2 Other #KSsinglecell highlights (subjective): STARsolo (10X faster mapper than Cell Ranger), Nichenet to model intercellular communication (<a href="https://t.co/1rgRRXEnCM" class="uri">https://t.co/1rgRRXEnCM</a>) and Dynbenchmark <span class="citation">@woutersaelens</span> - comparing dozens of sc trajectory inference methods <a href="https://t.co/PdVQSGfVyw" class="uri">https://t.co/PdVQSGfVyw</a>
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
Great talk on SPLIT-seq this morning at #KSsinglecell by <span class="citation">@seeliglab</span>. The power of this technique to democratize scRNAseq and improve experimental design via cost reduction is huge, but method has other technical perks too. We used their mouse data in our ALS study. Thanks!
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
bioigor
</td>
<td style="text-align:left;">
RAGE-seq: targeted long-read sequencing with short-read based transcriptome profiling to characterize full-length T cell (TCR) and B cell (BCR) receptor sequences and transcriptional profiles #KSSingleCell <a href="https://t.co/5k0SQHCWlI" class="uri">https://t.co/5k0SQHCWlI</a>
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Understanding spatial organisation of the genome is critical to decipher mechanisms and gene regulatory networks. Huge potential to apply mathematical models at the #singlecell resolution. Great talk from Caroline Uhler (MIT) #KSsinglecell
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Comp Bio Open panel discussion: What are the major upcoming challenges? <span class="citation">@dana_peer</span>: More mechanistic modeling by integrating additional data (ie. space, time) &amp;amp; understanding of technical and biological noise/signal #KSsinglecell
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Cool stuff from <span class="citation">@ManuSinghNZ</span> at #KSsinglecell. #RAGESeq: antigen receptor profiles by targeted full-length #scRNASeq with <span class="citation">@nanopore</span> sequencing. Highly accurate and cost-effective profiling of BCR and TCR sequences. <a href="https://t.co/7RNPOc4bN3" class="uri">https://t.co/7RNPOc4bN3</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Fascinating work from Barbara Treutlein from <span class="citation">@ETH_en</span>. scRNAseq profiles of developing and regenerating axolotl limbs show striking similarity. Multipotent blastema cells drive regeneration. Great system &amp;amp; study design! #KSsinglecell
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Powerful technique from <span class="citation">@AlejoFraticelli</span> and Fernando Camargo at #KSsinglecell. Iterative cell barcoding to track cell lineages and differentiation. Many potential applications in understanding stem cell biology. <a href="https://t.co/y5RYdYX9qX" class="uri">https://t.co/y5RYdYX9qX</a>
</td>
<td style="text-align:right;">
0.9937939
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
.@satijalab <span class="citation">@nygenome</span> Integrating different data types, all collected at single-cell resolution. Integration of sequencing data with imaging data particularly interesting, combines taxonomic classification with spatial information #KSSinglecell
</td>
<td style="text-align:right;">
0.9934318
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
tomkXY
</td>
<td style="text-align:left;">
Great to see <span class="citation">@MasahiroUeda13</span> <span class="citation">@SingleMolBio</span> our colleague from <span class="citation">@BDR_RIKEN</span> from Osaka, Japan present on live-cell imaging at #KSsinglecell. Non-invasive techniques for #singlecell analysis are essential to understand cell biology. 頑張って上田先生。お疲れ様です。
</td>
<td style="text-align:right;">
0.9953368
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
Amazing talk by <span class="citation">@lucaspelkmans</span> on crossing spatial scales using 4i (Iterative Indirect Immunofluorescence Imaging)! Subcellular spatial protein distribution in single cells provides another level of info on cellular states #KSSinglecell <span class="citation">@SystemsXch</span> <span class="citation">@UZH_en</span>
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
.@dana_peer Continuous view of #CellFate: modelling differentiation using Markov chains, defining for each cell the probability of reaching each terminal state. Application to mouse endoderm #SingleCell transcriptomes reveals unappreciated lineage relationships #KSSinglecell
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
Fascinating talk by Lucas Pelkmans about multiplexed protein maps, a way to image cells with subcellular resolution using common but unique multiplexed pixel profiles, <a href="https://t.co/ArfHjf1avy" class="uri">https://t.co/ArfHjf1avy</a>. I wonder how much more info this “dimension” will give to scRNAseq data #KSsinglecell
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
<span class="citation">@dana_peer</span>: “Cell fates can be modeled using markov chains to describe probabilities of cells ending up in terminal states.” Nice way to identify key regulators: Just associate gene expression with these probabilities. Prob better than using pseudotime(?) #KSsinglecell
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
Beautiful work from the <span class="citation">@AllenInstitute</span> presented by Hongkui Zheng at #KSsinglecell tonight. Retro-seq, patch-seq, multiplexed FISH, etc. Great to see the the promised advances in understanding the brain via sequencing and imaging technologies delivered. <a href="https://t.co/crw5WtLgeJ" class="uri">https://t.co/crw5WtLgeJ</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
ebfabyanic
</td>
<td style="text-align:left;">
Really happy to be here at #KSSingleCell 2019 in Breckenridge! I’m going to be giving a poster tonight (poster number 1054), and on Thursday, I’ll be giving a short talk during the “Novel Single Cell Technologies” workshop. Please stop by! I appreciate any and all feedback! 🧬 <a href="https://t.co/QXh4Zd31gd" class="uri">https://t.co/QXh4Zd31gd</a>
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
I’ll be presenting our study of molecular pathology in ALS tonight (poster 3004) at #KSsinglecell. In it we use Spatial Transcriptomics (now part of ⁦<span class="citation">@10xgenomics</span>⁩) to profile expression in ~1300 mouse tissue sections and 80 PM sections. Stop by! <a href="https://t.co/DI7mEtLqjh" class="uri">https://t.co/DI7mEtLqjh</a>
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
arnavm1
</td>
<td style="text-align:left;">
During one of the talks at #KSsinglecell, someone (can’t remember who-sorry! 😳) mentioned RFP toxicity. Reminded me of this paper (<a href="https://t.co/HtnCunOupO" class="uri">https://t.co/HtnCunOupO</a>) by David Piston’s lab evaluating toxicity of basically every known FP. Spoiler alert: GFP wins.
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
manimota
</td>
<td style="text-align:left;">
Amazing talk just given by ⁦<span class="citation">@lucaspelkmans</span>⁩ at #KSsinglecell. Highly multiplexed IF imaging reveals great wealth of information at subcellular to cell population scales. Even without the high multiplexing, the Ab elution technique is super useful! <a href="https://t.co/1eyaSQ32yf" class="uri">https://t.co/1eyaSQ32yf</a>
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
JCoolScience
</td>
<td style="text-align:left;">
Day 2 of #KSsinglecell off to a fast start. <span class="citation">@lucaspelkmans</span> kicked it off with a tour de force overview of 4i technology across biological scales. Ellen Rothenberg followed up with a beautiful dissection of T-cell differentiation mechanisms using single cell methods.
</td>
<td style="text-align:right;">
0.9951352
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
lkmklsmn
</td>
<td style="text-align:left;">
<span class="citation">@BartDeplancke</span> and his Techno team improve efficiency of droplet based scRNAseq using DisCo (deterministic co-encapsulation): Avoid incomplete droplets to increase cell capture efficiency and optimize bead recovery. <a href="https://t.co/jawQ2GbUlH" class="uri">https://t.co/jawQ2GbUlH</a> Quite promising! #KSsinglecell
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
GCLinderman
</td>
<td style="text-align:left;">
Stephen Quake at #KSSingleCell discusses an application of scRNA-seq for drug development: Cell atlases like Tabula Muris provide a resource to identify which tissues or cell types a potential drug target is expressed in.
</td>
<td style="text-align:right;">
0.9949154
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
bibaswanghoshal
</td>
<td style="text-align:left;">
The concept of gene regulation and causal inference is exciting! Soft interventions or knockdowns provide similar information on causal inference as the hard interventions or knockouts gives a different perspective to look at gene regulatory networks! #KSsinglecell
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
AlejoFraticelli
</td>
<td style="text-align:left;">
#KSsinglecell Keystone Single Cell Biology 2019 was a blast! My recap: 1) scRNAseq increasingly combined with other single cell technologies 2) scProteomics is almost here 3) scOmics data integration 4) We have a data storage problem 5) Visualization tools need much improvement
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
MolSystBiol
</td>
<td style="text-align:left;">
Great keynote by Scott Fraser on new approaches for multimodal-multiplex-multidimensional #imaging. Improving image/noise ratio, solving the “speed problem” &amp;amp; adding dimensions to imaging (time, gene, color, z) #KSSinglecell #microscopy
</td>
<td style="text-align:right;">
0.9946749
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
FanZhang_Jessie
</td>
<td style="text-align:left;">
I will give a talk on “Integrating Bulk and Single-Cell Transcriptomics with Mass Cytometry Data to Define Inflammatory Cell States in Rheumatoid Arthritis” on session of “Extracting Information from High-dimensional Measurements” #KSsinglecell <span class="citation">@KeystoneSymp</span>. Ask me questions!
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
fabian_theis
</td>
<td style="text-align:left;">
Rick Horwitz now talking at #KSsinglecell on large team-based effort of the <span class="citation">@AllenInstitute</span> for Cell Science. Last year they went through a “computational tool building phase - next year comes the fun”. Hey, we comp people have fun, too. You should hear our jokes.
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
BartDeplancke
</td>
<td style="text-align:left;">
With some delay: day 2 #KSsinglecell standouts (subjective!): the theme of cyclic immunofluorescence (4i (Gut et al., Science 2018; <span class="citation">@lucaspelkmans</span> ) with great application on intestinal organoids / superb imaging <span class="citation">@priscaliberali</span> &amp;amp; Immuno-SABER <a href="https://t.co/1JdoIAtC2A" class="uri">https://t.co/1JdoIAtC2A</a>
</td>
<td style="text-align:right;">
0.9944104
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
stchlk
</td>
<td style="text-align:left;">
Airline lost my poster in Denver. Took evasive action. Excited to be attending #KSsinglecell with poster copy number two. Ready to learn about exciting developments in single cell biology esp. bioinformatics
</td>
<td style="text-align:right;">
0.9941183
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">10</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">celda</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/campbio/celda" class="uri">https://github.com/campbio/celda</a></td>
</tr>
<tr class="even">
<td align="left">nichenetr</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/browaeysrobin/nichenetr" class="uri">https://github.com/browaeysrobin/nichenetr</a></td>
</tr>
<tr class="odd">
<td align="left">scRNA-tools</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/oshlack/scrna-tools" class="uri">https://github.com/oshlack/scrna-tools</a></td>
</tr>
<tr class="even">
<td align="left">too-many-cells</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/gregoryschwartz/too-many-cells" class="uri">https://github.com/gregoryschwartz/too-many-cells</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>##  setting  value                       
##  version  R version 3.5.1 (2018-07-02)
##  system   x86_64, darwin15.6.0        
##  ui       X11                         
##  language (EN)                        
##  collate  en_AU.UTF-8                 
##  tz       Australia/Melbourne         
##  date     2019-01-21                  
## 
##  package      * version date       source        
##  assertthat     0.2.0   2017-04-11 CRAN (R 3.5.0)
##  backports      1.1.2   2017-12-13 CRAN (R 3.5.0)
##  base         * 3.5.1   2018-07-05 local         
##  bindr          0.1.1   2018-03-13 CRAN (R 3.5.0)
##  bindrcpp     * 0.2.2   2018-03-29 CRAN (R 3.5.0)
##  broom          0.5.0   2018-07-17 CRAN (R 3.5.0)
##  cli            1.0.0   2017-11-05 CRAN (R 3.5.0)
##  colorspace     1.3-2   2016-12-14 CRAN (R 3.5.0)
##  compiler       3.5.1   2018-07-05 local         
##  crayon         1.3.4   2017-09-16 CRAN (R 3.5.0)
##  datasets     * 3.5.1   2018-07-05 local         
##  devtools       1.13.6  2018-06-27 cran (@1.13.6)
##  digest         0.6.16  2018-08-22 CRAN (R 3.5.0)
##  dplyr        * 0.7.6   2018-06-29 cran (@0.7.6) 
##  evaluate       0.11    2018-07-17 CRAN (R 3.5.0)
##  fansi          0.3.0   2018-08-13 CRAN (R 3.5.0)
##  forcats      * 0.3.0   2018-02-19 CRAN (R 3.5.0)
##  ggforce        0.1.3   2018-07-07 CRAN (R 3.5.0)
##  ggplot2      * 3.0.0   2018-07-03 CRAN (R 3.5.0)
##  ggraph       * 1.0.2   2018-07-07 CRAN (R 3.5.0)
##  ggrepel      * 0.8.0   2018-05-09 cran (@0.8.0) 
##  glue           1.3.0   2018-07-17 CRAN (R 3.5.0)
##  graphics     * 3.5.1   2018-07-05 local         
##  grDevices    * 3.5.1   2018-07-05 local         
##  grid           3.5.1   2018-07-05 local         
##  gridExtra      2.3     2017-09-09 CRAN (R 3.5.0)
##  gtable         0.2.0   2016-02-26 CRAN (R 3.5.0)
##  here         * 0.1     2017-05-28 CRAN (R 3.5.0)
##  highr          0.7     2018-06-09 CRAN (R 3.5.0)
##  htmltools      0.3.6   2017-04-28 CRAN (R 3.5.0)
##  httr           1.3.1   2017-08-20 CRAN (R 3.5.0)
##  igraph       * 1.2.2   2018-07-27 CRAN (R 3.5.0)
##  janeaustenr    0.1.5   2017-06-10 CRAN (R 3.5.0)
##  jsonlite       1.5     2017-06-01 CRAN (R 3.5.0)
##  knitr        * 1.20    2018-02-20 CRAN (R 3.5.0)
##  labeling       0.3     2014-08-23 CRAN (R 3.5.0)
##  lattice        0.20-35 2017-03-25 CRAN (R 3.5.1)
##  lazyeval       0.2.1   2017-10-29 CRAN (R 3.5.0)
##  lubridate    * 1.7.4   2018-04-11 CRAN (R 3.5.0)
##  magrittr       1.5     2014-11-22 CRAN (R 3.5.0)
##  MASS           7.3-51  2018-10-16 CRAN (R 3.5.0)
##  Matrix         1.2-14  2018-04-13 CRAN (R 3.5.1)
##  memoise        1.1.0   2017-04-21 CRAN (R 3.5.0)
##  methods      * 3.5.1   2018-07-05 local         
##  modeltools     0.2-22  2018-07-16 CRAN (R 3.5.0)
##  munsell        0.5.0   2018-06-12 cran (@0.5.0) 
##  nlme           3.1-137 2018-04-07 CRAN (R 3.5.1)
##  NLP            0.1-11  2017-08-15 CRAN (R 3.5.0)
##  parallel       3.5.1   2018-07-05 local         
##  pillar         1.3.0   2018-07-14 CRAN (R 3.5.0)
##  pkgconfig      2.0.2   2018-08-16 CRAN (R 3.5.0)
##  plyr           1.8.4   2016-06-08 CRAN (R 3.5.0)
##  purrr        * 0.2.5   2018-05-29 CRAN (R 3.5.0)
##  R6             2.2.2   2017-06-17 CRAN (R 3.5.0)
##  RColorBrewer * 1.1-2   2014-12-07 CRAN (R 3.5.0)
##  Rcpp           0.12.18 2018-07-23 CRAN (R 3.5.0)
##  reshape2       1.4.3   2017-12-11 CRAN (R 3.5.0)
##  rlang          0.2.2   2018-08-16 CRAN (R 3.5.0)
##  rmarkdown      1.10    2018-06-11 CRAN (R 3.5.0)
##  rprojroot      1.3-2   2018-01-03 CRAN (R 3.5.0)
##  rtweet       * 0.6.7   2018-06-20 CRAN (R 3.5.0)
##  scales         1.0.0   2018-08-09 CRAN (R 3.5.0)
##  slam           0.1-43  2018-04-23 CRAN (R 3.5.0)
##  SnowballC      0.5.1   2014-08-09 CRAN (R 3.5.0)
##  stats        * 3.5.1   2018-07-05 local         
##  stats4         3.5.1   2018-07-05 local         
##  stringi        1.2.4   2018-07-20 CRAN (R 3.5.0)
##  stringr      * 1.3.1   2018-05-10 CRAN (R 3.5.0)
##  tibble         1.4.2   2018-01-22 CRAN (R 3.5.0)
##  tidyr        * 0.8.1   2018-05-18 CRAN (R 3.5.0)
##  tidyselect     0.2.4   2018-02-26 CRAN (R 3.5.0)
##  tidytext     * 0.1.9   2018-05-29 CRAN (R 3.5.0)
##  tm             0.7-5   2018-07-29 CRAN (R 3.5.0)
##  tokenizers     0.2.1   2018-03-29 CRAN (R 3.5.0)
##  tools          3.5.1   2018-07-05 local         
##  topicmodels  * 0.2-7   2017-11-03 CRAN (R 3.5.0)
##  tweenr         0.1.5   2016-10-10 CRAN (R 3.5.0)
##  units          0.6-0   2018-06-09 CRAN (R 3.5.0)
##  utf8           1.1.4   2018-05-24 CRAN (R 3.5.0)
##  utils        * 3.5.1   2018-07-05 local         
##  viridis      * 0.5.1   2018-03-29 CRAN (R 3.5.0)
##  viridisLite  * 0.3.0   2018-02-01 cran (@0.3.0) 
##  withr          2.1.2   2018-03-15 CRAN (R 3.5.0)
##  wordcloud    * 2.6     2018-08-24 CRAN (R 3.5.0)
##  xml2           1.2.0   2018-01-24 CRAN (R 3.5.0)
##  yaml           2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
